generators/server/templates/src/main/java/package/config/_OAuth2ServerConfiguration.java

```diff
@@ -85,6 +91,8 @@ public void configure(HttpSecurity http) throws Exception {
             .and()
                 .csrf()
                 .disable()
+                // By default CORS is disabled. Uncomment here and in application.yml to enable.
+                // .addFilterBefore(corsFilter, UsernamePasswordAuthenticationFilter.class)
```

Comment:
We have very often the question to add CORS, so this should be easy to use. Isn't there a way to enable it only when it's configured? For example you test if the variable is null or not, and only add it when it's not.
I'll have a look if I have time this evening